test/run-gen-wasm.py

```diff
@@ -46,30 +46,38 @@ def main(args):
   parser.add_argument('-p', '--print-cmd', action='store_true',
                       help='print the commands that are run.')
   parser.add_argument('--no-debug-names', action='store_true')
-  parser.add_argument('--generate-names', action='store_true')
+  parser.add_argument('--objdump', action='store_true',
+                      help="objdump the resulting binary")
   parser.add_argument('file', help='test file.')
   options = parser.parse_args(args)
 
   gen_wasm = utils.Executable(sys.executable, GEN_WASM_PY,
                               error_cmdline=options.error_cmdline,
                               basename=os.path.basename(GEN_WASM_PY))
 
+  objdump = utils.Executable(
+      find_exe.GetWasmdumpExecutable(options.bindir),
+      error_cmdline=options.error_cmdline)
+
   wasm2wast = utils.Executable(
       find_exe.GetWasm2WastExecutable(options.bindir),
       error_cmdline=options.error_cmdline)
   wasm2wast.AppendOptionalArgs({
       '--no-debug-names': options.no_debug_names,
-      '--generate-names': options.generate_names,
   })
+  wasm2wast.AppendOptionalArgs({'--verbose': options.verbose,})
```

Comment:
No, I just moved this line up.  Not that --verbose is not the same as .verbose attribute below.